docs/quick-start/environment.mdx

```diff
@@ -25,7 +28,15 @@ In the second terminal, deploy the test contract:
 yarn deploy
 ```
 
-This command deploys a test smart contract to the local network. The contract is located in `packages/hardhat/contracts` and can be modified to suit your needs. The `yarn deploy` command uses the deploy script located in `packages/hardhat/deploy` to deploy the contract to the network. You can also customize the deploy script.
+This command deploys a test smart contract to the local network. The contract can be modified to suit your needs. Is located in:
```

Comment:
I feel like this sentence is a bit unclear, how about:

"This command deploys a test smart contract to the local network. The contract can be modified to suit your needs and can be found in:"